app/views/staff/shared/activities/_filter.html.haml

```diff
@@ -6,6 +6,6 @@
       .govuk-form-group
         %label{ class: "govuk-label", for: :organistaion_id }
           = t("filters.activity.organisation")
-        = select_tag :organisation_id, options_from_collection_for_select(Organisation.all, :id, :name, organisation_id), { class: "govuk-select" }
+        = select_tag :organisation_id, options_from_collection_for_select(Organisation.delivery_partners.order(:name), :id, :name, organisation_id), { class: "govuk-select" }
```

Comment:
Do we have access to the form helper, to use `list_of_delivery_partners` here? If not, is it worth consolidating the list of delivery partners to the application helper, so we don't scatter the definition around? Are there contexts where we *don't* want it sorted by name?

Reply:
Good call - I think ordering it by name aids findability in the other case too, so I'll add the order filter in the helper too